mongodbatlas/project_ip_whitelist.go

```diff
@@ -7,6 +7,13 @@ import (
 	"net/url"
 )
 
+/* IMPORTANT: DEPRECATION NOTICE
+Deprecated: Access List Replaces Whitelist
```

Comment:
[super-nit] can we move this below line 21? james cover how to do this in go but right now making this a global comment makes the whole package (`mongodbatlas`) deprecated, you want to flag that only the `ProjectIPWhitelistService` is the one being deprecated 

Same with `WhitelistAPIKeysService` and I think we can skip commenting on the test file

Reply:
Removed from tests and moved lower in latest commit.